src/services/defaultClasses.js

```diff
@@ -14,6 +14,9 @@ export default {
         'vue-ads-border-b': true,
         'vue-ads-italic': true,
     },
+    selection: {
+        'bg-indigo-200': true,
```

Comment:
Not sure how css classes with 'vue-ads-' prefix are generated, `bg-indigo-200` is taken straight from tailwind. Please fix as needed.

Reply:
Okay, now I understand. The prefix is configured in `tailwind.config.js`, and it works without the prefix only because `tailwind.min.css` is linked to in `index.html`. Fixed in 8b5dda7

B/w, I think linking `tailwind.min.css` in `index.html` is not needed.